### PR TITLE
Move special route publisher

### PR DESCRIPTION
**Pull request by KludgeKML**
Move special-route-publisher rake tasks into publishing-api, as per https://github.com/alphagov/special-route-publisher/issues/260 . This is the first half of the move suggested in that issue - simply moving the task from a separate repo into publishing-api, and converting it to use the in-built Command class calls rather than calling through the API. Once this is complete special-route-publisher can be retired, and at our leisure we can complete the second half of the suggestions in the issue (publishing the special routes if necessary on deploy rather than having to explicitly call the tasks).

https://trello.com/c/9rOxWNnD/396-retire-special-route-publisher

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

**Review by ChrisBAshton — APPROVED** [review]
LGTM 👍 thanks so much for this!

File: docs/admin-tasks.md [hunk 1]
```diff
@@ -96,6 +96,54 @@ bundle exec rake expanded_links:populate_by_document_type['document-type']
 bundle exec rake expanded_links:truncate
 ```
 
+## Publishing special routes
+
+Special routes are content items that have to be put into publishing-api / content-store so that routes
+can be created for them in router-api's database (this includes things like assets that have to be available
+at route or hardcoded pages). The routes are described in lib/data/special_routes.yaml and lib/data/homepage.yaml
```
[the remark this answers is not in the document]
Comment (KludgeKML) [reply docs/admin-tasks.md:103]: Ah yes, generally I think best to put documentation first in a PR, but in this case that does make it a bit wierd. I'll move it as I fix the changes.

[the remark this answers is not in the document]
Comment (KludgeKML) [reply docs/admin-tasks.md:103]: Done

File: lib/tasks/special_route_publisher.rb [hunk 2]
```diff
@@ -0,0 +1,107 @@
+module Tasks
```
[the remark this answers is not in the document]
Comment (KludgeKML) [reply lib/tasks/special_route_publisher.rb:1]: I was following the example of DataSanitizer (which has both the rake task and the class it uses in the tasks directory). To keep this tidy, should I make a PR where I move both classes into /lib?

File: lib/data/homepage.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+- :content_id: "f3bbdec2-0e62-4520-a7fd-6ffd5d36e03a"
```
[the remark this answers is not in the document]
Comment (KludgeKML) [reply lib/data/homepage.yaml:1]: Hmm. It was a separate file in special-route-publisher, but yes, it needn't be, we could exclude it by document type.

File: content_schemas/formats/shared/definitions/publishing_app.jsonnet [hunk 4]
```diff
@@ -28,7 +28,7 @@
       "share-sale-publisher",
       "short-url-manager",
       "smartanswers",
-      "special-route-publisher",
+      "publishing-api",
```
[the remark this answers is not in the document]
Comment (KludgeKML) [reply content_schemas/formats/shared/definitions/publishing_app.jsonnet:31]: Oh yes, that's my bad.

**Pull request merged**